lib/game.rb

```diff
@@ -42,7 +44,13 @@ def alert_current_player
     console.print_message("It is #{@current_player}\'s turn")
   end
 
+  def winner?
```

Comment:
It looks like you're using "?" to indicate "may return a null value" whereas it's conventional in ruby for "?" to mean "returns a boolean value".

Reply:
Yes, you are correct. For some reason I was thinking it was a boolean result until I just looked at it in IRB.... totally nil 🙈